front_end/panels/ai_chat/LLM/OpenRouterProvider.ts

```
@@ -324,7 +344,7 @@ export class OpenRouterProvider extends LLMBaseProvider {
       const response = await fetch(this.getToolSupportingModelsEndpoint(), {
         method: 'GET',
         headers: {
-          'Authorization': `Bearer ${this.apiKey}`,
+          'Authorization': `Bearer ${this.getApiKey()}`,
```

ESLint

[error] Unnecessarily quoted property 'Authorization' found.

(@stylistic/quote-props)